src/services/todos.ts

```diff
@@ -0,0 +1,14 @@
+import todosFromServer from '../api/todos';
+import { Todo } from '../types/Todo';
+import { getUserById } from './user';
+
+export function getPreparedTodos() {
+  return todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
```

Comment:
Ensure that `getUserById` returns a valid user object. Consider handling cases where a user might not be found to prevent potential runtime errors.